_examples/hello-webrpc-ts/webapp/src/client.gen.ts

```diff
@@ -89,7 +89,6 @@ export class ExampleService implements ExampleService {
       ).then((res) => {
       return buildResponse(res).then(_data => {
         return {
-        
```

Comment:
The extra empty line was fixed by https://github.com/webrpc/gen-typescript/commit/f516a621bb3a0281db8de2d908a8c1e30abe9664